### DISCUSSION TITLE
[zodios-openapi] status code descriptions

### DISCUSSION THREAD

**Discussion by epotter2297** [ecyrbe/zodios#214] [Ideas]
Right now, the default status code description is either `Success` or `Not Found`. I'm not sure if this would be too prescriptive or not, but it would be cool to have the defaults be what is specified [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).

On a related note, there also isn't a way to give custom descriptions to responses that aren't errors (If I wanted `201` to have a description of `Created`, I can't do that at the moment)

**Comment by ecyrbe** [comment]
Would you be willing to provide a PR, this should be easy.
- On `@zodios/core` you could add an optional attribute `responseDescription` to `ZodiosEndpointDefinition` in `zodios.types.ts`
- Use it on `@zodios/openapi`, replace `Success` with `endpoint.responseDescription ?? 'Success'`

Else i'll add this when i have time (working on something else)

**Reply by epotter2297** [reply]
Sure! I should have some free time in the next day or two.

**Comment by ecyrbe** [comment]
@epotter2297 it's available in `@zodios/core` v10.5.2 and `@zodios/openapi` v10.4.5

**Reply by epotter2297** [reply]
Aww you beat me to it lol
